securedrop/tests/test_crypto_util.py

```diff
@@ -23,3 +24,23 @@ def test_clean(self):
             crypto_util.clean('foo bar`') # backtick is not currently allowed
         with self.assertRaises(crypto_util.CryptoException):
             crypto_util.clean('bar baz~') # tilde is not currently allowed
+
+    def test_encrypt(self):
+        journalist, _ = utils.db_helper.init_journalist()
+        source, _ = utils.db_helper.init_source()
+        replies = utils.db_helper.reply(journalist, source, 1)
```

Comment:
Why are you creating a `Reply` here? Think this should probably just be removed.